Makefile

```diff
@@ -8,6 +8,7 @@ mypy:
 	mypy --ignore-missing-imports $(MODULES)
 
 test: lint mypy
+	tests/fixtures/populate.py --s3-bucket $(DSS_S3_BUCKET_TEST_FIXTURES) --gs-bucket $(DSS_GS_BUCKET_TEST_FIXTURES)
```

Comment:
Test methods are sorted alphanumerically. IMHO you could just test PUT, then test GET which happens to depend on PUT working. It's not ideally decomposed, but I don't see a major problem with that 😁